packages/admin-panel/src/widgets/InputField.js

```diff
@@ -196,6 +206,24 @@ export const InputField = ({
         />
       );
       break;
+    case 'link': {
+      const { path, parameters = {} } = linkOptions;
+      let link = path;
+      Object.entries(parameters).forEach(([paramKey, recordProperty]) => {
+        const linkParam = recordData[recordProperty];
+        if (linkParam) {
+          link = link.replace(`:${paramKey}`, linkParam);
+        }
```

Comment:
This is to construct the Route path `viz-builder/:visualisationId`, so that we can pass the id of exisiting viz to the viz-builder page